src/components/ResultForm.jsx

```diff
@@ -0,0 +1,47 @@
+import "../components/ResultForm.css";
+
+const ResultForm = ({ arrQuestion = [], arrAnswers = [], timeLong }) => {
+  let score = 0;
+  const result = arrAnswers.map((answer, index) => {
+    const isCorret = arrQuestion[index].answers[answer].isCorrect;
+    isCorret ? score++ : null;
+    return (
+      <>
+        <div>
```

Comment:
It's a good idea to extract the iterated JSX into a separate component.

Reply:
In each iteration I calculate the score and create the result. At the moment, I think there is no need to create a new component and pass everything used here. And also score is needed before generating the whole result (as a component). Or what idea you would have for it?